backend/pkg/git/config.go

```diff
@@ -0,0 +1,32 @@
+package git
+
+import (
+	"flag"
+)
+
+// Config for Git Service
+type Config struct {
+	// There might come more repositories (e. g. protobuf?) which we want to serve
+	TopicDocumentationRepo RepositoryConfig `yaml:"topicDocumentation"`
+
```

Comment:
todo: open an issue so we don't forget to add a config option to allow raw html (gets escaped currently)